administrator-manual/en/webmail.rst

```diff
@@ -65,4 +65,16 @@ If you want to open the access from any network: ::
 
   config setprop roundcubemail access public
   signal-event nethserver-roundcubemail-update
+  
+Removing
+========
+
+You can remove Roundcube to add another webmail type (eg: SOGo, WebTop.…) and you can keep all the mail server RPMs.
+
+To do this only remove Roundcube from the Installed list in the Software Center.
+
+If you want remove Roundcube, remove all the mail server RPMs and no longer have an email system on the server, first remove Roundcube from the Installed list in the Software Center and when it's completed do this step on the server console – either directly or by ssh. ::
+
+   yum autoremove
```

Comment:
I thought autoremove removes all unused dependencies.  If they are not needed, does it matter if you do "autoremove" or "autoremove nethserver-roundcubemail"?

Also I check by testing them out and this is the results.

The first one is by command "yum autoremove nethserver-roundcubemail".  Removing 1 package + 93 dependents.

![screenshot from 2016-11-22 08-33-00]

(https://cloud.githubusercontent.com/assets/23269582/20535522/ed3abd7e-b099-11e6-9c12-95aedc654ffd.png)

The second one is by removing only nethserver-roundcubemail:  Only removing 1 package.

![screenshot from 2016-11-22 09-03-02](https://cloud.githubusercontent.com/assets/23269582/20535908/c208266c-b09b-11e6-8e84-decb23538a28.png)

The third on is using the command "yum autoremove":

![screenshot from 2016-11-22 08-58-02](https://cloud.githubusercontent.com/assets/23269582/20535645/7a16f4ce-b09a-11e6-9863-b2c80899051e.png)

They are the same.  "autoremove nethserver-roundcubemail" removes a total of 94 packages.  "remove netheserver-roundcubemail" removes 1 and "autoremove" removes 4 + 89 = 94.